assets/textures/default_ceil.xpm

```diff
@@ -0,0 +1,14 @@
+/* XPM */
+static char * wall_xpm[] = {
```

Comment:
This is the same name of the file
```suggestion
static char * default_ceil[] = {
```

Reply:
These textures are just temporary and will never be displayed. We will replace them of course